spec/spec_helper.rb

```diff
@@ -5,9 +5,10 @@
 
 CENTOS_7 = {
   platform: 'centos',
-  version: '7.2.1511',
+  version: '7.4.1708',
 }.freeze
 
+# Is this line necessary? Seems to repeat line 9.
 ALL_PLATFORMS = [
   CENTOS_7,
 ].freeze
```

Comment:
`ALL_PLATFORMS` is a list of the different platforms so in ChefSpec we can iterate over the list and run the tests once for each platform. An example of this in this cookbook is: https://github.com/osuosl-cookbooks/ibm-power/blob/18be4aa6c155d1bc38ac317335b8c65a7175a27a/spec/unit/recipes/ohai_plugin_spec.rb#L4-L8

I think we can leave this in since we might eventually want to support both CentOS 7 and future versions like CentOS 8 simultaneously; if we replaced `ALL_PLATFORMS` with `CENTOS_7` it would mean doing more work if we decided we wanted to go back and support multiple platforms again.

Reply:
Ok got it. For some reason I was assuming all_platforms was re-categorizing the platforms as having the "freeze" thing, not creating a variable for all platforms.